### DISCUSSION TITLE
A bug in followers number

### DISCUSSION THREAD

**Discussion by ghost** [community/community#113585] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

It states that I have five followers on the page for my profile. Meanwhile, there is only one follower visible when I click on it.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Chipsum ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by jfullstackdev** [comment]
yup even in my Incognito, I can confirm this is an irregular behavior, though if a person's account is deleted, it went to ghost account, not sure what it looks like if he made the account private

![image](https://github.com/community/community/assets/47092464/482f04f0-1cc5-4f7f-b7ee-db1355340845)

**Comment by ghost** [comment]
@jdevstatic Exactly, probably private/ghost account, should we make this resolved or we wait for a concrete confirmation ?

**Reply by jfullstackdev** [reply]
it's now up to the GitHub Staff in here to decide
